prow/plugins/releasenote/releasenote.go

```diff
@@ -39,8 +39,8 @@ const (
 	// merge.
 	ReleaseNoteLabelNeeded    = "do-not-merge/release-note-label-needed"
 	releaseNote               = "release-note"
-	releaseNoteNone           = "release-note-none"
-	releaseNoteActionRequired = "release-note-action-required"
+	releaseNoteNone           = "release-note/none"
+	releaseNoteActionRequired = "release-note/action-required"
```

Comment:
So as far as supporting both labels at the same time goes, the pattern I saw when `mungegithub` had to do this in the past was keep the old label name around in vars prefixed with `deprecated`, eg:`deprecatedReleaseNoteNone` and `deprecatedReleaseNoteActionRequired`

then anything that _read_ the label checked for both old and new, and anything that _wrote_ the label only used the new label

Reply:
Doesn't `label_sync` have support for migrating labels? I don't think we need to do that